python/cross_service/photo_analyzer/README.md

```diff
@@ -0,0 +1,169 @@
+#  Analyzing photos using Amazon Rekognition with the AWS SDK for Python
+
+## Overview
+
+Shows you how to use the AWS SDK for Python (Boto3) to create a web application that 
+lets you upload photos to an Amazon Simple Storage Service (Amazon S3) bucket, use 
+Amazon Rekognition to analyze and label the photos, and use Amazon Simple Email 
+Service (Amazon SES) to send email reports of image analysis.  
```

Comment:
Shows you how to use the AWS SDK for Python (Boto3) to create a web application that 
lets you do the following: 

- Upload photos to an Amazon Simple Storage Service (Amazon S3) bucket.
- Use Amazon Rekognition to analyze and label the photos. 
- Use Amazon Simple Email Service (Amazon SES) to send email reports of image analysis.

Reply:
done